content/intro-to-storybook/react-native/pt/simple-component.md

```diff
@@ -0,0 +1,187 @@
+
+Agora vamos criar o componente de tarefa e o arquivo de história que o acompanha: `components/Task.jsx` e `components/Task.stories.jsx`.
+
+Começaremos com uma implementação básica da `Task`, simplesmente pegando os atributos que sabemos que precisaremos e as duas ações que você pode realizar em uma tarefa (para movê-la entre as listas):
+
+```jsx:title=components/Task.jsx
+import { StyleSheet, TextInput, View } from "react-native";
+import { styles } from "./styles";
+
+export const Task = ({
+  task: { id, title, state },
+  onArchiveTask,
+  onPinTask,
+}) => {
+  return (
+    <View style={styles.listItem}>
+      <TextInput value={title} editable={false} />
+    </View>
+  );
+};
+```
+
+Agora adicione o arquivo da história:
+
+```jsx:title=components/Task.stories.jsx
+import { Task } from "./Task";
+
+export default {
+  title: "Task",
+  component: Task,
+  argTypes: {
+    onPinTask: { action: "onPinTask" },
+    onArchiveTask: { action: "onArchiveTask" },
+  },
+};
+
+export const Default = {
+  args: {
+    task: {
+      id: "1",
+      title: "Test Task",
+      state: "TASK_INBOX"
+    },
+  },
+};
+
+export const Pinned = {
+  args: { task: { ...Default.args.task, state: "TASK_PINNED" } },
+};
+
+export const Archived = {
+  args: { task: { ...Default.args.task, state: "TASK_ARCHIVED" } },
+};
+```
+
+Em nossos arquivos de histórias, usamos uma sintaxe chamada Component Story Format (CSF). Neste caso, estamos usando o CSF3, que é uma nova versão atualizada do CSF ​​compatível com as versões mais recentes do Storybook. Nesta versão do CSF, há muito menos clichê, o que facilita o início.
+
+There are two basic levels of organization in Storybook: the component and its child stories. Think of each story as a permutation of a component. You can have as many stories per component as you need.
+
+Existem dois níveis básicos de organização no Storybook: o componente e suas histórias filhas. Pense em cada história como uma permutação de um componente. Você pode ter quantas histórias por componente precisar.
+
+- **Componente**
+  - História
+  - História
+  - História
+
+Para informar ao Storybook sobre o componente que estamos documentando, criamos uma exportação padrão `default` que contém:
+
+- `component` -- o próprio componente
+- `title` -- como se referir ao componente na barra lateral do aplicativo Storybook
+- `argTypes` -- nos permite especificar os tipos de nossos args, aqui usamos para definir ações que serão registradas sempre que essa interação ocorrer
+
+Para definir nossas histórias, exportamos um objeto com uma propriedade `args`. Argumentos ou [`args`](https://storybook.js.org/docs/react/writing-stories/args) para abreviar, nos permitem editar ao vivo nossos componentes com o complemento de controles sem reiniciar o Storybook. Uma vez que um valor [`args`](https://storybook.js.org/docs/react/writing-stories/args) muda, o mesmo acontece com o componente.
```

Comment:
@Wanderson77 "controles" doesn't exist in the Portuguese language and should be reverted back to its English form as we're referring to the actual Controls addon.